content/handbook/security/corporate/end-user-services/supported-apps/google-workspace/google-mail.md

```
@@ -0,0 +1,89 @@
+translated_at: "2026-05-11T00:00:00Z"
+translator: claude
+stale: false
+---
+
+Gmail は私たちの主要なメールアプリケーションです。Okta ダッシュボードの `G Suite SSO Mail` タイル経由、または直接 [gmail ページ](https://mail.google.com/) にアクセスすることで、アプリにアクセスできます。
+
+このページは、これまでに受けた最もよくある質問のいくつかへの解決策を提供することを目的としています。より詳細なサポートについては、[公式 Gmail サポートページ](https://support.google.com/mail/) をご覧ください。
+
+## 目次
+
+- [Gmail アクセスリクエスト](#gmail-access-requests)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)